README.md

```diff
@@ -133,6 +137,75 @@ Assets and events will ignore existing ids on deletes. If you prefer to abort th
 when an attempt to delete an unknown id, use `.option("ignoreUnknownIds", "true")`
 for those resources types.
 
+## Asset hierarchy (beta)
+Note: The asset hierarchy utility is currently in beta, and has not been sufficiently tested to be used 
```

Comment:
Maybe change "The asset hierarchy utility" to "Asset hierarchy ingestion"? Not sure though, I think it's fine as is really.

Reply:
I was conflicted here as well, what should we call it? Ingestion sounds like it's a first time thing, while the tool is actually most useful when there are updates.

Reply:
I agree, "ingestion" is not the right word here. I've heard the term "asset syncing" a lot, but I'm not sure if that's the right term either. I'll leave it up to you.